muspinsim/input/keyword.py

```diff
@@ -89,7 +89,11 @@ def __init__(self, block=[], args=[]):
             )
 
         use_default = False
-        if len(block) == 0 and self.has_default:
```

Comment:
just to check, is this removal of `self.has_default` done because every keyword has a default now?

Reply:
I missed that - probably best to check if the keyword has a default or throw an error